Source/WebCore/css/parser/CSSSelectorParser.cpp

```diff
@@ -601,15 +599,16 @@ bool CSSSelectorParser::consumeName(CSSParserTokenRange& range, AtomString& name
 
     if (range.peek().type() != DelimiterToken || range.peek().delimiter() != '|')
         return true;
-    range.consume();
 
     namespacePrefix = name;
-    const CSSParserToken& nameToken = range.consume();
-    if (nameToken.type() == IdentToken) {
-        name = nameToken.value().toAtomString();
-    } else if (nameToken.type() == DelimiterToken && nameToken.delimiter() == '*')
+    if (range.peek(1).type() == IdentToken) {
+        range.consume();
+        name = range.consume().value().toAtomString();
+    } else if (range.peek(1).type() == DelimiterToken && range.peek(1).delimiter() == '*') {
+        range.consume();
+        range.consume();
```

Comment:
Copy paste?

Reply:
Well, this was called twice intentionally, consuming '|' and then '*' sequentially.

Reply:
Ignore my comment. 👍